android/sdk/src/main/java/org/jitsi/meet/sdk/audiomode/AudioModeModule.java

```diff
@@ -300,7 +300,6 @@ private boolean updateAudioRoute(int mode) {
             audioManager.setMode(AudioManager.MODE_NORMAL);
             audioManager.abandonAudioFocus(null);
             audioManager.setSpeakerphoneOn(false);
-            audioManager.setMicrophoneMute(true);
```

Comment:
maybe it's worth to leave that commented out with comment above(PR description), so that in future nobody will try to add it back